backend/src/app/models/group.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from app.models.base_model import BaseModel
+
+class GroupModel(BaseModel):
+    STATUS_CHOICES = (
+        ('ACTIVE', 'Active'),
+        ('INACTIVE', 'InActive'),
+    )
+
+    name = models.CharField(verbose_name='Group Name', max_length=225, unique=True)
```

Comment:
where is company id, How you will know which group belongs to which company?  Also unique key must be on company id and name.